wagtail/admin/views/home.py

```diff
@@ -352,6 +353,7 @@ def default(request):
     raise Http404
 
 
+icon_comment_pattern = re.compile(r"<!--.*?-->")
```

Comment:
This will match all comments, not just the ones correctly using `!`. This saves about 10% of the size of the sprite (before gzip, so not much for correctly set up site, but lots for local dev).